.github/workflows/deploy-tutorials.yml

```
@@ -0,0 +1,38 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+name: Deploy tutorials
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        description: Where to upload the tutorials. Must start with "Learning platform".
```

Any reason to not explicitly say the two options?

```suggestion
        description: Where to upload the tutorials. Either "Learning platform (staging)" or "Learning platform (production)".
```